nl_server/routes.py

```diff
@@ -35,7 +35,8 @@
 
 @bp.route('/healthz')
 def healthz():
-  default_index_type = current_app.config[config.DEFAULT_INDEX_TYPE_KEY]
+  default_index_type = current_app.config[
+      config.EMBEDDINGS_SPEC_KEY]['default_index']
```

Comment:
is it `defaultIndex`?  Use constants?

Reply:
no, EmbeddingsSpec object uses default_index. updated to constants

Reply:
Hmm, if its a class, why aren't we using `.default_index` ?

Reply:
you're right, updated